17.py

```diff
@@ -0,0 +1,117 @@
+class Manager:
+    def __init__(self):
+        self.menu = Menu()
+        self.notes_submanager = NotesSubManager()
+        self.cards_submanager = CardsSubManager()
+
+    def start(self):
+        while True:
+            self.welcome_screen()
+            choice = self.menu.get_choice()
+            self.execute(choice)
+
+    def welcome_screen(self):
+        self.menu.show_menu()
+
+    def execute(self, choice):
+        match choice:
+            case 1:
+                self.notes_submanager.add()
+            case 2:
+                self.cards_submanager.add()
+            case 3:
+                self.show_notes()
+            case 4:
+                self.show_cards()
+            case 5:
+                print("Zakończono.")
+                exit()
+            case _:
+                print("Nieprawidłowy wybór. Spróbuj ponownie.")
+
+    def show_notes(self):
+        self.notes_submanager.show()
+
+    def show_cards(self):
+        self.cards_submanager.show()
+
+
+class Menu:
+    def show_menu(self):
+        print("""\
+        Menu:
+        1. Dodaj notatkę
+        2. Dodaj wizytówkę
+        3. Wyświetl wszystkie notatki
+        4. Wyświetl wszystkie wizytówki
+        5. Wyjdź
+        """)
+
+    def get_choice(self):
+        try:
+            choice = int(input("Którą opcję Pan/Pani wybiera?: "))
+            return choice
+        except ValueError:
+            raise ValueError("Nieprawidłowy wybór. Wprowadź liczbę.")
```

Comment:
to wygląd jak wiadomość dla użytkownika (logika biznesowa) i nie powinien to być wyjątek